src/game/objects/har.c

```diff
@@ -2024,9 +2024,6 @@ void har_finished(object *obj) {
         h->state = STATE_DONE;
         har_set_ani(obj, ANIM_VICTORY, 0);
     } else if(h->state == STATE_VICTORY || h->state == STATE_DONE) {
-        // end the arena
-        DEBUG("ending arena!");
-        game_state_set_next(obj->gs, SCENE_MENU);
```

Comment:
Something somewhere else still handles this ?

Reply:
arena_end handles this.